test/Microsoft.OpenAPI.OData.Reader.Tests/Resources/TripService.OpenApi.yaml

```diff
@@ -5777,7 +5777,7 @@ paths:
           content:
             application/json:
               schema:
-                title: Collection of Person
+                title: Collection of Trip
```

Comment:
I believe that part of the scope was to make sure this becomes a referenced schema instead of inline to avoid duplication (the collection is defined multiple times here)